pkg/cluster/nodes/node.go

```diff
@@ -224,3 +224,35 @@ func (n *Node) WriteKubeConfig(dest string) error {
 
 	return ioutil.WriteFile(dest, buff.Bytes(), 0600)
 }
+
+// WaitForReady uses kubectl inside the "node" container to check if the
+// master nodes are "Ready".
+func (n *Node) WaitForReady(nodeName string, until time.Time) bool {
```

Comment:
let's make this a function on Context, and check for all nodes to be ready?

Reply:
all nodes matching the selector*

Reply:
@BenTheElder  The function relies on `tryUntil` from pkg/cluster/nodes, so not importable.
What if we added it as a utility function in pkg/cluster/nodes/nodes.go?
Change function sig to `func WaitForReady(node *nodes.Node, until time.Time) bool`?

Reply:
That makes sense to me. We can always move it later as well.